tests/test_csp.py

```diff
@@ -13,7 +13,7 @@ def test_csp_assign():
     assignment = {}
     australia.assign(var, val, assignment)
 
-    assert australia.nassigns == 1
+    # assert australia.nassigns == 1
```

Comment:
Why was this commented out? I am not familiar with AC-3, but generally removing tests is not good.

Reply:
I'm sorry, I added the test again! :+1: